README.md

```diff
@@ -171,6 +171,74 @@ this is loaded as it starts.
+config :grizzly,
+  run_zipgateway_bin: false
+```
+
+#### Configuring zipgateway
+
+
+The `zipgateway` binary is passed a configuration file named `zipgateway.cfg`. This has configuration
+parameters around networking and setting device specific information.
+
+Supported configuration fields are:
+
+* `:unsolicited_destination_ip6` -  IPv6 string address defaults to `"fd00:aaaa::2` and should work for
+  normal Grizzly use cases.
+* `:unsolicited_destination_port` - a port for the DTLS server to use (default `41230`),
+* `:ca_cert` - a path to the CA certificate of the portal provider (default priv dir of Grizzly)
+* `:cert` - a path to the Z/IP Gateway public certificate (defeault priv dir of Grizzly)
+* `:priv_key` - a path to the Z/IP Gateway private key (defeault priv dir of Grizzly)
+* `:eeprom_file` - a path to the eeprom file. The eeprom file must be stored in
+   a non-volatile location.
+   (default priv dir of Grizzly)
+* `:tun_script` - a path to the `.tun` script (default priv dir of Grizzly)
+* `:pvs_storage_file` - the storage file for Smart Start provisioning data. This
+   file should be located on a writable
+   non-volatile partition. The file will be generated if it does not exist
+   (default `/root/provisioning_list_store.dat`)
+* `:provisioning_config_file` - The Smart Start provisioning list initial
+   configuration file. This file is an initialization file, it will only
+   be read if the provisioning list store file does not exist
+   (default `/etc/zipgateway_provisioning_list.cfg`)
+* `:pan_ip6` - IPv6 prefix of the Z-Wave network. All Z-Wave nodes will be
+   assigned an IPv6 address in this prefix (default `"fd00:bbbb::1`)
+* `:lan_ip6` - The IPv6 address of the Z/IP Gateway. This is LAN IP6 address of
+   the Z/IP Gateway. This is the address which should be used when communicating
+   with the Z/IP Gateway (default `"fd00:bbbb::1`)
+* `:lan_gw6` - he IPv6 default gateway which the Z/IP Gateway will use, when sending
+   outside the LAN or PAN (default `"::1"`)
+* `:psk` - Pre shared key used in DTLS connection (default `"123456789012345678901234567890AA"`)
+* `:manufacturer_id`: Id to set in the version report (default `0`)
+* `:hardware_version` - Hardware version to set in the version report (default `1`)
+* `:product_id` - Id to set in the version report (default `1`)
+* `:product_type` - Id to set in the version report (default `1`)
+* `:serial_log` - Log file for serial communication. Used for debugging. If this
+   option is not set the no logging is done (default none)
+
+For the most part if you are using Grizzly to run zipgateway the defaults should
+just work. You will mostly just need configure manufacturer id, product id and type,
+and hardware version.
+
+```elixir
+config :grizzly,
+  zipgateway_cfg: %{
+    manufacturer_id: 123,
+    product_type: 123,
+    product_id: 123,
+    hardware_version: 123
+  }
+```
+
+Where `123` is just place holder and you should change out for your real information.
```

Comment:
While I doubt that whomever was assigned `123` will notice, it's probably better for the `manufacturer_id` to be set to 0 in the example to avoid stomping on anyone. You might also want to mention that manufacturer IDs are given out by Silicon Labs and aren't needed for nearly anything that you'd use Grizzly for.